pkg/cmd/condition/delete.go

```diff
@@ -63,24 +63,51 @@ or
 				return err
 			}
 
-			return deleteConditions(s, p, args)
+			return deleteConditions(s, p, args, opts.DeleteAllNs)
 		},
 	}
 	f.AddFlags(c)
 	c.Flags().BoolVarP(&opts.ForceDelete, "force", "f", false, "Whether to force deletion (default: false)")
+	c.Flags().BoolVarP(&opts.DeleteAllNs, "all", "", false, "Delete all Conditions in a namespace (default: false)")
 	_ = c.MarkZshCompPositionalArgumentCustom(1, "__tkn_get_condition")
 	return c
 }
 
-func deleteConditions(s *cli.Stream, p cli.Params, condNames []string) error {
+func deleteConditions(s *cli.Stream, p cli.Params, condNames []string, deleteAll bool) error {
 	cs, err := p.Clients()
 	if err != nil {
 		return fmt.Errorf("failed to create tekton client")
 	}
 	d := deleter.New("Condition", func(conditionName string) error {
 		return cs.Tekton.TektonV1alpha1().Conditions(p.Namespace()).Delete(conditionName, &metav1.DeleteOptions{})
 	})
+	if deleteAll {
+		condNames, err = allConditionNames(p, cs)
+		if err != nil {
+			return err
+		}
+	}
 	d.Delete(s, condNames)
-	d.PrintSuccesses(s)
+
+	if !deleteAll {
+		d.PrintSuccesses(s)
+	} else if deleteAll {
+		if d.Errors() == nil {
+			fmt.Fprintf(s.Out, "All Conditions deleted in namespace %q\n", p.Namespace())
```

Comment:
`has been` ?

Reply:
This follows the language used in every pr for `--all`. The language also is past tense and brief, so I would prefer to keep it unless you feel strongly.

Reply:
np!